index.html

```diff
@@ -10,6 +10,68 @@
 </head>
 <body>
   <h2 class="title">Front-End Tools</h2>
-  <h2 class="title">Back-End Tools</h2>
+  <div class="boxes">
+    <div class="box">
+      <img class="box-cover" src="assets/react_icon.svg" alt="">
+        <div class="box-title">ReactJS </div>
```

Comment:
use semantic tags like headlines, paragraphs, labels or spans for text containers. this would be a good headline with a lower hierarchy than the title h2. I'd use an h4 here.

Reply:
remember to use 2 space indentation, this is indented inside the image and that's not right.